src/composables/useHotKey/index.ts

```diff
@@ -124,16 +126,31 @@ export function useHotKey(
 
 	/**
 	 * Validates event key to expected key
-	 * FIXME should support any languages / key codes
 	 *
 	 * @param event keyboard event
 	 * @param key expected key
 	 * @return whether it satisfies expected value or not
 	 */
 	const validateKeyEvent = (event: KeyboardEvent, key: string): boolean => {
-		if (options.caseSensitive) {
-			return event.key === key
+		// If key exactly matches event.key, valid with any caseSensitive option. Do not perform further checks
+		if (event.key === key) {
+			return true
+		}
+
+		const isKeyInLowerCase = key === key.toLowerCase()
+		const isEventKeyInLowerCase = event.key === event.key.toLowerCase()
+
+		// If key and event.key are in different cases, invalid with caseSensitive = true. Do not perform further checks
+		if (options.caseSensitive && isKeyInLowerCase !== isEventKeyInLowerCase) {
+			return false
```

Comment:
maybe do not compute this all the time?

```suggestion
		// If key and event.key are in different cases, invalid with caseSensitive = true. Do not perform further checks
		if (options.caseSensitive) {
			const isKeyInLowerCase = key === key.toLowerCase()
			const isEventKeyInLowerCase = event.key === event.key.toLowerCase()
 			if (isKeyInLowerCase !== isEventKeyInLowerCase) {
				return false
			}
```